example-app/SantokuApp/src/features/qa-home/pages/HomePage.tsx

```diff
@@ -156,8 +156,8 @@ export const HomePage: React.VFC<HomePageProps> = ({
   );
   const tagIconColor = useMemo(() => (selectedTagId ? 'blue' : 'black'), [selectedTagId]);
 
-  const scrollViewRef = useRef<ScrollView>();
-  const scrollToTop = useCallback(() => scrollViewRef.current?.scrollTo({y: 0, animated: true}), []);
+  const flatListRef = useRef<FlatList>(null);
+  const scrollToTop = useCallback(() => flatListRef.current?.scrollToOffset({offset: 0, animated: true}), []);
```

Comment:
Why is this operation limited to Android?

Reply:
In iOS, the scrolling moves to the top when the status bar is tapped by an OS feature.
Therefore, only Android is supported by the application.